fastlane/Fastfile

```diff
@@ -53,21 +74,22 @@ platform :ios do
   desc "Publish a beta release to internal TestFlight testers"
   lane :beta do
     signin
-    increment_build_number({
-      build_number: latest_testflight_build_number + 1,
-      xcodeproj: "NAMS.xcodeproj"
-    })
+    increment_build_number(
+      {
+        build_number: latest_testflight_build_number + 1
+      }
+    )
     build
     commit = last_git_commit
-    upload_to_testflight(
-      distribute_external: true,
-      groups: [
-        "External Testers"
-      ],
-      submit_beta_review: true,
-      notify_external_testers: true,
-      expire_previous_builds: true,
-      changelog: commit[:message]
-    )
+    # upload_to_testflight(
+    #   distribute_external: true,
+    #   groups: [
+    #     "External Testers"
+    #   ],
+    #   submit_beta_review: true,
+    #   notify_external_testers: true,
+    #   expire_previous_builds: true,
+    #   changelog: commit[:message]
+    # )
   end
```

Comment:
@PSchmiedmayer Also, the actual upload to TestFlight is not yet enabled? Can I just enable that or do I have to consider anything there?